evaluation/FEEDBACK.md

```diff
@@ -0,0 +1,61 @@
+# Feedback
+
+This document is aiming to aggregate a set of generic rules to make writing good feedback a little easier. Note that it is by far not covering everything.
+
+P.S. Don't forget that common sense goes above all guidelines. Be smart.
+
+## Purpose
+
+Each point in feedback should be aimed to helping receiving person improve. Think about it when writing each of the points.
```

Comment:
"in feedback" -> "of the feedback"
"aimed to helping" -> "aimed at helping" or "aimed to help"
"helping receiving person improve" -> "helping the receiving person improve"